src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
It's the figure object. Subplots return a figure and axes. In this case, the figure is unused, while the axis is passed in to heatmap.  The history on this particular choice is interesting:  https://docs.google.com/document/d/11u6uLNBzlveZVPkBADjvTi9f9m0Y7VIhzy4WOlWFbaY/edit#bookmark=id.kcj0bnkkqg39